jetstream/analysis.py

```diff
@@ -33,7 +60,7 @@ class Analysis:
     config: AnalysisConfiguration
 
     def __attrs_post_init__(self):
-        self.logger = logging.getLogger(__name__)
+        self.logger = logger
```

Comment:
We may as well just have everything use the module logger and drop `Analysis.logger`. I think I was worried that if we switched to a multiprocess model that it would be bad to use a logger that was created in the parent process but I'm not sure that's a real concern.